src/main/webapp/app/exam/participate/summary/exercises/programming-exam-summary/programming-exam-summary.component.ts

```diff
@@ -15,7 +15,5 @@ export class ProgrammingExamSummaryComponent {
     @Input()
     participation: ProgrammingExerciseStudentParticipation;
 
-    @Input()
-    submission: ProgrammingSubmission;
```

Comment:
Also remove the import to prevent warnings

Reply:
> The summary page itself work fine, this will be very useful.
> 
> One thing I noticed:
> A split second before the summary appears after pressing the button, this shows up:
> ![image](https://user-images.githubusercontent.com/11130248/86541458-72386500-bf0d-11ea-9c91-40fa9e8bb05a.png)
> In the console, I get
> 
> Log Entry
> This seems to be the case for all exams, but I guess this is related to the working time form.

I already fixed this, the problem was that the button was in the same form as the save working time button

Reply:
fixed